nixos/modules/services/networking/tailscale.nix

```diff
@@ -75,6 +81,13 @@ in {
       stopIfChanged = false;
     };
 
+    boot.kernel.sysctl = mkIf (cfg.useRoutingFeatures == "server" || cfg.useRoutingFeatures == "both") {
```

Comment:
Can you make it so the individual options are `mkDefault true` rather than `true`?
Otherwise, it would prevent fine-grained `net.ipv4.conf.$interface.forwarding = true`.

BTW, I think it would be better to have by-default fine-grained option, but I do not have enough Tailscale knowledge / context to make propositions.

Reply:
Updated :+1: